docs/gc.md

```diff
@@ -132,15 +132,35 @@ There are a number of advantages to using arraylets.
 
 ### `metronome` policy
 
-[`-Xgcpolicy:metronome`](xgcpolicy.md#metronome-aix-linux-x86-only) is designed for applications that require precise response times. Garbage collection occurs in small interruptible steps to avoid stop-the-world pauses. This policy is available only on x86 Linux and AIX platforms.
+**(Linux on x86-64 and AIX platforms only)**
 
-### `optavgpause` policy
+[`-Xgcpolicy:metronome`](xgcpolicy.md#metronome-aix-linux-x86-only) is designed for applications that require a precise upper bound on collection pause times as well as specified application utilization: the proportion of time that the application is permitted to use, with the remainder being devoted to GC. The realtime GC runs in short interruptible bursts to avoid stop-the-world pauses.
 
-[`-Xgcpolicy:optavgpause`](xgcpolicy.md#optavgpause) uses concurrent mark and sweep phases, which means that pause times are reduced when compared to optthruput, but at the expense of some performance throughput.
+The realtime heap is allocated as a contiguous range of RAM partitioned into small regions of equal size, usually about 64 KB. 
 
-### `optthruput` policy
+As with the `balanced` GC policy, arrays are represented as arraylets with a spine pointing to a series of regions containing the array elements (see [Arraylets: dealing with large arrays](#arraylets-dealing-with-large-arrays) for more information). Note that JNI access to array data might involve reconstituting arraylets as contiguous arrays, which can significantly slow down processing.
 
-[`-Xgcpolicy:optthruput`](xgcpolicy.md#optthruput) is optimized for throughput by disabling the concurrent mark phase, which means that applications will stop for long pauses while garbage collection takes place. You might consider using this policy when high application throughput, rather than short garbage collection pauses, is the main performance goal.
+Each region of the heap is either empty or contains only objects of the same size or an arraylet. This organization simplifies new object allocation and consolidation of free heap space, allowing the realtime collector to maintain GC throughput while supporting a consistent service level.
```

Comment:
more precisely, it's same _size class_

There is a lower and upper bound for the size (upper bound of one size class is same as the lower bound of the next large size class. Next size class (their bounds) is approx 20% larger than previous.

The point is not in allocation simplification nor consolidation of free space, but in
1) reducing heap fragmentation, so that need for heap compaction/de-fragmentation is minimized
2) more precise control of incremental sweep phase

Reply:
Updates made.